presto-main/src/main/java/io/prestosql/sql/planner/planprinter/TextRenderer.java

```diff
@@ -132,6 +131,7 @@ private String printStats(PlanRepresentation plan, NodeRepresentation node)
         output.append(format(", Output: %s (%s)\n", formatPositions(nodeStats.getPlanNodeOutputPositions()), nodeStats.getPlanNodeOutputDataSize().toString()));
 
         printDistributions(output, nodeStats);
+        printCollisions(output, nodeStats);
```

Comment:
Maybe add a test with EXPLAIN ANALYZE over JOIN, and over AGG,
verifying "Collisions" is in the output ?